packages/grafana-prometheus/src/result_transformer.ts

```diff
@@ -236,7 +237,8 @@ export function transformDFToTable(dfs: DataFrame[]): DataFrame[] {
       length: timeField.values.length,
     };
   });
-  return frames;
+
+  return frames.map((frame) => maybeSortFrame(frame, 0));
```

Comment:
Unfortunately this doesn't work for the escalation because it has 690 sorted values before it has values out of order, so `isLikelyAscendingVector` evaluates to true https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/transformations/transformers/joinDataFrames.ts#L85

Reply:
isLikelyAscendingVector has a sample size parameter, but it's probably not worth exposing here. we can just do a thorough loop here and call sortDataFrame() if anything is out of order

Reply:
switched to sorting those frames where a pushed timestamp is ever < previous timestamp